public/app/features/panel/panel_header.ts

```diff
@@ -12,7 +12,10 @@ const template = `
   <span class="panel-time-info" ng-if="ctrl.timeInfo"><i class="fa fa-clock-o"></i> {{ctrl.timeInfo}}</span>
 </span>`;
 
-function renderMenuItem(item, ctrl) {
+function renderMenuItem(
+  item: { divider: any; submenu: any; click: any; href: any; icon: any; text: any; shortcut: any },
```

Comment:
there is a PanelMenuItem type from @grafana/ui you can use

Reply:
Not sure how to handle this. The properties being used are `divider, submenu, click, href, icon` while  `PanelMenuItem` look like
```typescript

export interface PanelMenuItem {
  type?: 'submenu' | 'divider';
  text?: string;
  iconClassName?: string;
  onClick?: () => void;
  shortcut?: string;
  subMenu?: PanelMenuItem[];
}
```
Is the right way to go to add the fields being used to the type, or should the code be rewritten to match the type (which will probably break things)?

Reply:
Ok, can you create a new type named AngularPanelMenuItem for this then?